package.json

```diff
@@ -1,25 +1,26 @@
 {
-  "name": "eslint-config-nhds",
+  "name": "lint-config-nhds",
```

Comment:
RTFM but yea that works I'll change that up.

This does raise an issue, if we have two separate packages are we keeping them in sync as far as version? or version each one separately? I liked the idea of the github releases matching up with our package version but we could just version our entire style-guide and each subpackage seperately:

e.g.
```yaml

github-release: 1.0.0
eslint-version: 2.0.1
sass-lint-version: 1.9.0

```